### PR TITLE
Gas limit buffer

### PR DESCRIPTION
**Pull request by RanCohenn**
#411

### CODE REVIEW

**Review by ashachaf — APPROVED** [review]

File: src/services/web3/config.ts [hunk 1]
```diff
@@ -4,6 +4,8 @@ import { Token } from 'services/observables/tokens';
 import { calculatePercentageChange } from 'utils/formulas';
 import { get7DaysAgo } from 'utils/pureFunctions';
 import { EthNetworks } from './types';
+import { BigNumber } from 'bignumber.js';
+import { BigNumberish } from 'ethers';
```
Comment (zavelevsky) [inline src/services/web3/config.ts:8]: I see BigNumberish imported but no new code of it used

Comment (RanCohenn) [reply src/services/web3/config.ts]: Fixed

File: src/elements/modalVbnt/ModalVbnt.tsx [hunk 2]
```diff
@@ -45,9 +43,8 @@ export const ModalVbnt = ({
 }: ModalVbntProps) => {
   const { account, chainId } = useWeb3React();
   const [amount, setAmount] = useState('');
-  const [amountUSD, setAmountUSD] = useState('');
   const percentages = useMemo(() => [25, 50, 75, 100], []);
-  const [selPercentage, setSelPercentage] = useState<number>(-1);
+  const [_, setSelPercentage] = useState<number>(-1);
```
Comment (zavelevsky) [inline src/elements/modalVbnt/ModalVbnt.tsx:47]: I believe you can leave this empty - no need for the `_`

**Review by zavelevsky — APPROVED** [review]
LGTM

**Pull request merged**